holidays/countries/argentina.py

```diff
@@ -159,13 +159,14 @@ def _add_movable(hol_date: date, hol_name: str) -> None:
                 - If THU/FRI - observed on next MON
             """
             if self.observed:
-                if self._is_tuesday(hol_date) or self._is_wednesday(hol_date):
-                    hol_date = _get_nth_weekday_from(-1, MON, hol_date)
-                    hol_name = self.tr("%s (Observado)") % self.tr(hol_name)
-                elif self._is_thursday(hol_date) or self._is_friday(hol_date):
-                    hol_date = _get_nth_weekday_from(1, MON, hol_date)
-                    hol_name = self.tr("%s (Observado)") % self.tr(hol_name)
-            self._add_holiday(hol_name, hol_date)
+                observed_label = self.tr("(Observado)")
+                if self._is_tuesday(dt) or self._is_wednesday(dt):
+                    dt = _get_nth_weekday_from(-1, MON, dt)
+                    name = f"{self.tr(name)} {observed_label}"
```

Comment:
There are languages in which the "observed template" differs from "{name} {observed_label}". For example, in Thai it's `"ชดเชย{name}"`. So, I think it's better to keep template with `%s` placeholder.

Reply:
I didn't touch the Thai's placeholder as it hadn't produced any warnings upon `make l10n` run.
Could you add more details on why it's better to keep the placeholder for the countries changed in this PR?

Reply:
I mean, we won't be able to localize these countries into Thai correctly.

Reply:
I've adjusted the i18n implementation, PTAL.